src/components/Users/UsersList.jsx

```diff
@@ -59,45 +64,41 @@ const UsersList = ({ loadingActive }) => {
           <tr>
             <th>First Name</th>
             <th>Last Name</th>
-            <th>Pod URL</th>
+            <th>Web Id</th>
```

Comment:
This field was labeled Pod URL, but was actually displaying their Web ID. Updating label to reflect the data being displayed.